example-compose.yaml

```diff
@@ -3,32 +3,21 @@ services:
   watchtower:
     image: v2tec/watchtower
     restart: always
-    command: --label-enable --cleanup --interval 300 
+    command: --label-enable --cleanup --interval 300
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
   ssgbot:
-    image: superserious/bot
+    image: superserious/bot:latest
     # for dev
     # build: .
-    # volumes:
-    #   - ./:/code/
-    env_file: 
+    volumes:
+      - ./:/code/
```

Comment:
This is for the sqlite db right? i think its better to have a different directory name for it